src/ActionModal/utils/ActionManager.js

```diff
@@ -77,16 +77,11 @@ export default class ActionManager {
 
     const command = payload.simulate ? "estimate" : "broadcast"
 
-    // TODO refactor and put into config.js
-    const graphqlHost = urlParams =>
-      (urlParams.graphql ? decodeURIComponent(urlParams.graphql) : false) ||
-      config.graphqlHost
-    const urlParams = getURLParams(window)
+    const graphqlHost = getGraphqlHost()
 
-    return fetch(
-      `${graphqlHost(urlParams)}/transaction/${command}`,
-      options
-    ).then(r => r.json())
+    return fetch(`${graphqlHost}/transaction/${command}`, options).then(r =>
```

Comment:
what is `r`? can you use the whole word?

Reply:
@iambeone you missed this comment. Do you check your GitHub notifications?

Reply:
Yes, I saw it, but forgot to react. I will be more careful

Reply:
changed it